google_gemini/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "93179a9e-98f8-48fe-843a-59f9c9bb84df",
+  "app_id": "google-gemini",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Optimize Google Gemini usage: cost estimates, prompt sampling and performance metrics.",
```

Comment:
@sriramkraman Due to lack of copy instructions. I duplicated the same description on Openai let me know if you need this changed